benches/hierarchy/engine_factory.rs

```diff
@@ -32,7 +32,7 @@ impl EngineFactory<BTreeEngine> for BTreeEngineFactory {
 }
 
 impl fmt::Debug for BTreeEngineFactory {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
You get used to it :-) It's actually useful because it tells you that `Formatter` contains borrowed data which you wouldn't know otherwise. It will also become a hard error not to have it at some point.

Reply:
Useful + 1